pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -479,6 +479,10 @@ type GlobalSMTPConfig struct {
 	// Note that Go does not support unencrypted connections to remote SMTP endpoints.
 	// +optional
 	RequireTLS *bool `json:"requireTLS,omitempty"`
+
+	// he default TLS configuration for SMTP receivers
```

Comment:
Hi @dongjiang1989 I'm guessing a typo
```suggestion
	// The default TLS configuration for SMTP receivers
```

Reply:
Thanks @heliapb 
It's indeed a typo.
Fixed